### ISSUE TITLE
Banner bypassed extension on several sites

### ISSUE THREAD

**Issue by sakuragasaki46** [MathRobin/Never-Consent#63]
On December 16, 2020, [on systutorials.com](https://www.systutorials.com/docs/linux/man/3-XkbKeyTypesForCoreSymbols/), a GDPR banner appeared anyway.

As a detail, it did not have any "Reject all" option, but did have "Manage my choices" and "Agree to selected". I'll update you for any site further breaking the extension.

Would you additionally add a feedback link on extension description page?

**Comment by sakuragasaki46** [comment]
[iedunote.com](https://www.iedunote.com/how-to-create-a-custom-fonts) broke the extension too

**Comment by MathRobin** [comment]
Many thanks for your contribution!

Sadly, lot of websites have their own ribbon a'd my extension is currently only for third party platforms. I have some code for standalone solutions but it's clearly not the best way to solve the problem.

Will probably add the code for the sites you have listed here, but it will probably need a lot of maintenance. Will see. 

For any other new sites, don't add to this issue please. Will be hard to follow what is managed or not, feel free to create a new issue each time please! 

Many thanks again! 

Ps: if you have any idea on how to manage properly legacy banners, don't hesitate to share your idea(s). I'm stucked on this major problem
